jaxrs/src/main/java/feign/jaxrs/JAXRSContract.java

```diff
@@ -44,10 +41,12 @@ protected MethodMetadata parseAndValidateMetadata(Class<?> targetType, Method me
     return super.parseAndValidateMetadata(targetType, method);
   }
 
-  @Override
-  protected void processAnnotationOnClass(MethodMetadata data, Class<?> clz) {
-    Path path = clz.getAnnotation(Path.class);
-    if (path != null && !path.value().isEmpty()) {
+  public JAXRSContract() {
+    super.registerClassAnnotation(Path.class, (path, data) -> {
+      if (path.value().isEmpty()) {
```

Comment:
`Path` cannot have an empty value, by specification.  this should be an error.

Reply:
`checkState` instead?

Will do the chance.

Reply:
I restored the original code, for now won't try to enforce the contract around this.